src/components/UserDetailSections/PatronBlock/PatronBlock.js

```diff
@@ -212,10 +218,23 @@ class PatronBlock extends React.Component {
     } = this.state;
     let contentData = patronBlocks.filter(p => moment(moment(p.expirationDate).endOf('day')).isSameOrAfter(moment().endOf('day')));
     contentData = _.orderBy(contentData, ['metadata.createdDate'], ['desc']);
+
+    const columnMapping = {
+      [PATRON_BLOCKS_COLUMNS.type]: <FormattedMessage id="ui-users.blocks.columns.type" />,
+      [PATRON_BLOCKS_COLUMNS.displayDescription]: <FormattedMessage id="ui-users.blocks.columns.desc" />,
+      [PATRON_BLOCKS_COLUMNS.blockedActions]: <FormattedMessage id="ui-users.blocks.columns.blocked" />,
+    };
+
+    const columnWidths = {
+      [PATRON_BLOCKS_COLUMNS.type]: '100px',
+      [PATRON_BLOCKS_COLUMNS.displayDescription]: '350px',
+      [PATRON_BLOCKS_COLUMNS.blockedActions]: '250px',
+    };
```

Comment:
Maybe would be better to move  `columnMapping` and `columnWidths` to constructor from 'render' method?